Week1/homework/style.css

```diff
@@ -1 +1,81 @@
-/* add your styling here */
\ No newline at end of file
+  body {
+    background-color:  #212b37;
+    padding: 0px 0px 10px 0;
+    margin: auto;
+  }
+  
+  h1 {
+    text-align: center;
+    font-weight: bold;
+    font-size: 3em;
+    color: white;
+    font-family: Georgia, serif;
+    
+  }
+  
+  h2 {
+    text-align: center;
+    font-family: Impact, Charcoal, sans-serif;
+    font-size: 1.7em;
+    font-weight: normal;
+  }
+  
+  p {
+    font-family: "Comic Sans MS", cursive, sans-serif;
```

Comment:
As a side note, and maybe you already know this. But Comic Sans MS is normally not chosen as a font-family on websites. Furthermore I would not use 3 different font-families on a webpage.